packages/terra-form-select/src/_Frame.jsx

```diff
@@ -375,14 +387,18 @@ class Frame extends React.Component {
     ]);
 
     return (
+      // This warns that a combobox role requires a tab index.
+      // The linter is unable to detect the tab index returned from a function.
+      /* eslint-disable-next-line jsx-a11y/aria-activedescendant-has-tabindex */
       <div
         {...customProps}
         role="combobox"
-        aria-controls={this.state.isOpen ? 'terra-select-dropdown' : undefined}
+        aria-controls={this.state.isOpen ? 'terra-select-menu' : undefined}
```

Comment:
This all looks good to me. I am unfamiliar with aria-controls, aria-owns, and aria-activedescendant properties. Are these values associated with a class name or are they ready by a screen reader and need translated?

Reply:
I found this really helpful as a reference: https://www.w3.org/WAI/GL/wiki/Using_aria-activedescendant_to_allow_changes_in_focus_within_widgets_to_be_communicated_to_Assistive_Technology

Reply:
Thanks for the info! Interesting stuff. This seems like good information we should add for quick reference for other people.